src/StockportGovUK.NetStandard.Extensions.csproj

```diff
@@ -4,13 +4,15 @@
     <PackageId>StockportGovUK.NetStandard.Extensions</PackageId>
     <Title>A package that provides shared logic for shared models for StockportGovUK applications</Title>
     <TargetFramework>netstandard2.1</TargetFramework>
-    <VersionPrefix>1.0.7</VersionPrefix>
+    <VersionPrefix>1.0.9</VersionPrefix>
```

Comment:
As we are adding in new functionality we would bump the minor version not the patch version. The new version number should be 1.1.0